src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.Node.cs

```diff
@@ -72,5 +78,87 @@ private string GetDebuggerDisplay()
                 return NameSpan + ", " + ParentIndex;
             }
         }
+
+        public readonly struct ParameterTypeInfo
+        {
+            public string Name { get; }
+            public bool IsComplexType { get; }
```

Comment:
nit: consider just readonly fields.  
doc these with examples.

Reply:
this still applies.

Reply:
done